docs/source/flow-project.rst

```diff
@@ -214,7 +214,7 @@ As shown before, all *eligible* operations can then be executed with:
 
     ~/my_project $ python project.py run
 
-We can now parallelize the determination of projet status with processes. This can be done be setting a value for the ``flow.status_parallelization`` configuration as ``process``. Altogether there are 3 valid values for this, ``threads``, ``none``, or ``processes``, where ``threads`` being the default one which uses multithreading to perform parallelization and none leads to no parallelization at all.
+We can now parallelize the determination of projet status with processes. This can be done by setting a value for the ``flow.status_parallelization`` configuration as ``process``. Altogether there are 3 valid values for this, ``threads``, ``none``, or ``processes``, where ``threads`` being the default one which uses multithreading to perform parallelization and ``none`` leads to no parallelization at all.
```

Comment:
Documentation always describes the status quo and should only refer to historical states in exceptional circumstances where it has a lot of educational value for current users. Think of the software as a picture that you describe, you don't need to reference what the picture looked like while it was painted. The only place where you explicitly describe the evolution of the software is the changelog.
```suggestion
The status determination is by default parallelized with threads, however this can be turned off or switched to using processes by setting a value for the ``flow.status_parallelization`` configuration key to either ``none``, ``processes`` with ``threads`` being the implicit default value.
```
Please also provide either an example for setting that key from the CLI or link to a relevant section in the documentation that shows how to configure signac (flow).

Reply:
I have provided an example on how to configure signac flow [here](https://github.com/glotzerlab/signac-docs/pull/87/files#diff-430c8afc00a0685ddc648d3827cb1d44R223). Are you referring to the same or something different?

Reply:
No, I was referring to that, so that's sufficient.